src/lib_c/i686-linux-musl/c/sys/limits.cr

```diff
@@ -0,0 +1,3 @@
+lib LibC
+  NGROUPS_MAX = 32 # max supplemental group id's
```

Comment:
this doesn't look right

Reply:
Line 51
git.musl-libc.org/cgit/musl/tree/include/limits.h